### PR TITLE
Events page

### LINKED ISSUE #35: Task: Add events page
/events page to see the latest community events information with registration link

### PR DESCRIPTION
**Pull request by bharattech**
Events page

fixes #35

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -24,24 +24,35 @@ section = "blog"
...
+
 [[extra.list]]
 title = "Rust Resources"
 content = 'Get all the resources for you to get started from basics to advanced topics'
+url = "#"
 
 [[extra.list]]
 title = "Sub Communities"
 content = 'Information and PoC of communities near you to take part in and geek out on Rust together'
+url = "/docs/sub-communities/details"
```
Comment (me-diru) [inline content/_index.md:41]: thanks for adding this! can you also point to Events at "https://rustindia.github.io/docs/sub-communities/details/". 

continue the paragrph with ". Do checkout the latest [events conducted by these communities.](/events)"

File: templates/events/section.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+              top: 0;
+              left: 0;
+              bottom: 0;
+              right: 0;
+              width: 100%;
+              height: 100%;
+            }
+          </style>
+          <iframe scrolling="no" class="responsive-iframe"
+            src="https://serverless-events-zkwagd06.fermyon.app/static/events.html" title="rust-india-events" ></iframe>
```
Comment (me-diru) [inline templates/events/section.html:44]: really nice way to show the details, did you try adding Rust Kerala and Rust Chandigarh to the table? They seem to be missing

**Review by me-diru — COMMENTED** [review]
Overall lgtm! Thanks for the PR :D

I have added my thoughts

Please clean the commit history as well

File: content/_index.md (hunk 1 above)
Comment (bharattech) [reply content/_index.md:41]: No problem, Yes.

File: templates/events/section.html (hunk 2 above)
Comment (bharattech) [reply templates/events/section.html:44]: fixed the Rust Chandigarh

**Comment by bharattech** [comment]
to clean the commit history

**Pull request closed without merging**